packages/loader/driver-utils/src/blobCacheStorageService.ts

```diff
@@ -17,12 +19,9 @@ export class BlobCacheStorageService extends DocumentStorageServiceProxy {
         super(internalStorageService);
     }
 
-    public async read(id: string): Promise<string> {
+    public async readBlob(id: string): Promise<ArrayBufferLike> {
         const blob = this.blobs.get(id);
-        if (blob !== undefined) {
-            return blob;
-        }
-
-        return this.internalStorageService.read(id);
```

Comment:
This may already have been discussed in this PR. But why is the call to this.internalStorageService.read(id) removed?